kvmagent/kvmagent/plugins/hygon_device_plugin.py

```
@@ -0,0 +1,550 @@
+            This command:
+                - Removes all mdev devices created under /sys/bus/mdev/devices/
+                - Cleans up CCP device virtualization resources
+                - Should be called before detaching or reconfiguring CCP devices
+
+        Response Format:
+            {
+                "success": true,
+                "error": ""
+            }
+        """
+        cmd = jsonobject.loads(req[http.REQUEST_BODY])
```

Ruff (0.14.7)

Local variable `cmd` is assigned to but never used

Remove assignment to unused variable `cmd`

(F841)